server/api/cvrs.py

```diff
@@ -20,6 +21,37 @@
 )
 from ..util.csv_download import csv_response
 from ..util.csv_parse import decode_csv_file
+from ..util.jsonschema import JSONDict
+
+
+def set_contest_metadata_from_cvrs(contest: Contest):
+    contest.num_winners = 1  # TODO how do we get this from the CVRs?
+    contest.total_ballots_cast = 0
+
+    for jurisdiction in contest.jurisdictions:
+        cvr_contests_metadata = typing.cast(
+            JSONDict, jurisdiction.cvr_contests_metadata
+        )
+        contest_metadata = cvr_contests_metadata[contest.name]
+        if contest_metadata is None:
+            raise Conflict("Some jurisdictions haven't uploaded their CVRs yet.")
+
+        if not contest.choices:
+            contest.choices = [
+                ContestChoice(
+                    id=str(uuid.uuid4()),
+                    contest_id=contest.id,
+                    name=choice_name,
+                    num_votes=0,
+                )
+                for choice_name in contest_metadata["choices"]
+            ]
+
+        contest.total_ballots_cast += contest_metadata["total_ballots_cast"]
+        contest.votes_allowed = contest_metadata["votes_allowed"]
```

Comment:
does this get overwritten as the loop goes over the various jurisdictions?

Reply:
yeah, the other option i could think of was just to take the first jurisdiction's value and trust that - didn't seem any different from taking the last jurisdiction's value (by overwriting it each iteration). thoughts?